src/builder/QuarkBuilder.sol

```diff
@@ -982,6 +997,317 @@ contract QuarkBuilder {
         });
     }
 
+    struct MorphoBorrowIntent {
+        uint256 amount;
+        string assetSymbol;
+        uint256 blockTimestamp;
+        address borrower;
+        uint256 chainId;
+        uint256 collateralAmount;
+        string collateralAssetSymbol;
+    }
+
+    function morphoBorrow(
+        MorphoBorrowIntent memory borrowIntent,
+        Accounts.ChainAccounts[] memory chainAccountsList,
+        PaymentInfo.Payment memory payment
+    ) external pure returns (BuilderResult memory) {
+        List.DynamicArray memory actions = List.newList();
+        List.DynamicArray memory quarkOperations = List.newList();
+
+        bool useQuotecall = false; // never use Quotecall
```

Comment:
iirc there were some prior discussion about this. The conclusion that was reached was that we instead declare `useQuotecall` on top, to avoid having a plain `false` in later quark actions function calls creating confusions.   

So actually it was moved from having just `false` in each functions to declare this on top as `useQuotecall`